ra-optimisation/linear-range-action-rao/src/test/java/com/farao_community/farao/linear_range_action_rao/LinearRangeActionRaoTest.java

```diff
@@ -77,15 +85,21 @@ public void getVersion() {
     }
 
     @Test
-    public void run() {
+    public void runTest() {
         Network network = Importers.loadNetwork(
                 "TestCase12Nodes.uct",
                 getClass().getResourceAsStream("/TestCase12Nodes.uct")
         );
         Crac crac = create();
-
         String variantId = "variant-test";
-
+        Map<State, SensitivityComputationResults> stateSensiMap = new HashMap<>();
+        Map<Cnec, Double> cnecMarginMap = new HashMap<>();
+        crac.getCnecs().forEach(cnec -> cnecMarginMap.put(cnec, 1.0));
+        Map<Cnec, Double> cnecMaxThresholdMap = new HashMap<>();
+        crac.getCnecs().forEach(cnec -> cnecMaxThresholdMap.put(cnec, 500.));
+
+        PowerMockito.mockStatic(SystematicSensitivityAnalysisService.class);
```

Comment:
if you do this i think you can remove the `MockSensitivityComputationFactory` below in this class

Reply:
ok